src/peft/peft_model.py

```diff
@@ -1784,9 +1737,7 @@ def forward(
             inputs_embeds = torch.cat((prompts, inputs_embeds), dim=1)
             return self.base_model(inputs_embeds=inputs_embeds, **kwargs)
 
-    def _cpt_forward(
-        self, input_ids=None, inputs_embeds=None, peft_config=None, task_ids=None, batch_size=None, **kwargs
-    ):
+    def _cpt_forward(self, input_ids, inputs_embeds, peft_config, task_ids, batch_size, **kwargs):
```

Comment:
Why this change?

Reply:
I don't remember my reasoning back then, but checking this now, it wouldn't make sense to use the default arguments, e.g.

`device = [i.device for i in [input_ids, inputs_embeds, labels] if i is not None][0]`

would fail if the arguments were `None`.